### ISSUE TITLE
JSON Lines serializer tests are unreliable

### ISSUE THREAD

**Issue by dmajda** [juttle/juttle#152]
I often get this test failure locally when running `gulp test` with Node.js 4.2.4:

```
  1) serializers/jsonl can write no points to a provided stream:
     Error: timeout of 10000ms exceeded. Ensure the done() callback is being called in this test.
```

Deeper in the log, there is this error:

```
  serializers/jsonl
    ✓ can instantiate a jsonl serializer
[16:22:51] 'test' errored after 44 s
[16:22:51] Error in plugin 'gulp-mocha'
Message:
    ENOENT: no such file or directory, open '/var/folders/zp/7fg2ysbj4g9drhgxdd32dk2c0000gn/T/tmp-31317JQgtQ5Zfaqxi'
Details:
    errno: -2
    code: ENOENT
    syscall: open
    path: /var/folders/zp/7fg2ysbj4g9drhgxdd32dk2c0000gn/T/tmp-31317JQgtQ5Zfaqxi
    domain: [object Object]
    domainThrown: true
Stack:
Error: ENOENT: no such file or directory, open '/var/folders/zp/7fg2ysbj4g9drhgxdd32dk2c0000gn/T/tmp-31317JQgtQ5Zfaqxi'
    at Error (native)
    at Object.fs.openSync (fs.js:549:18)
    at Object.fs.readFileSync (fs.js:397:15)
    at /Users/dmajda/Jut/Code/juttle/test/adapters/serializers/jsonl.spec.js:25:23
    at doNTCallback0 (node.js:419:9)
    at process._tickDomainCallback (node.js:389:13)
    1) can write no points to a provided stream
    ✓ can write out a few points correctly
```

**Comment by dmajda** [comment]
This also happens on Travis CI, see e.g. [this job](https://travis-ci.org/juttle/juttle/jobs/101880046).

@rlgomes Could you have a look?

**Comment by rlgomes** [comment]
I'll have a look at this today

**Comment by dmajda** [comment]
Fixed by #157.
